src/Writers/html.jl

```diff
@@ -0,0 +1,27 @@
+"""
+Provides the `render()` methods to write the documentation as HTML files
+(`MIME"text/html`).
```

Comment:
Missing a closing `"`. Can you also escape the opening and closing `"` so that GitHub doesn't mangle the highlighting? With `MIME\"text/html\"`. Same with the others like this.